api/src/data/seeds/014.data-ingestion-sources.ts

```diff
@@ -0,0 +1,41 @@
+import knex from "knex";
+import { DataIngestionSource } from "src/data/models";
+
+export async function seed(knex: knex.Knex) {
+  const data_ingestion_sources = await knex<DataIngestionSource>("data_ingestion_sources");
+
+  const toInsert = [
+    {
+      source_name: "RL6",
```

Comment:
Consider using the code from WRAP https://github.com/icefoganalytics/wrap/blob/9da5e31931351ce911d441556caa388d4dc2ac35/api/src/db/db-client-legacy.ts#L48 and https://github.com/icefoganalytics/wrap/blob/9da5e31931351ce911d441556caa388d4dc2ac35/api/src/db/db-client-legacy.ts#L60 so that you can use camelCase in JS instead of having a mix of camelCase and snake_case.

Docs are at https://knexjs.org/guide/#postprocessresponse